test

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 
 echo "Running tests..."
-gb test
+go test
```

Comment:
This was using [gb](https://getgb.io/) for its vendoring and tests, I migrated it to Glide as I was running into problems getting gb working and to compile.  If we would prefer we can revert back to gb and revert these changes.